<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicermiAnnotationModuleAnnotationPropertyDialog</class>
 <widget class="QDialog" name="qSlicermiAnnotationModuleAnnotationPropertyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>660</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="qCTKCollapsibleGroupBox" name="displayPropertiesCTKCollapsibleGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>281</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Display Properties</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <widget class="QTabWidget" name="propertyTabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>281</width>
      <height>301</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="textTab">
     <attribute name="title">
      <string>Text</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>18</y>
        <width>251</width>
        <height>104</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="qCTKColorPickerButton" name="textUnselectedColorPickerButton">
           <property name="text">
            <string>Selected Text Color</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="qCTKColorPickerButton" name="textSelectedColorPickerButton">
           <property name="text">
            <string>Unselected Text Color</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="textScaleLabel">
           <property name="text">
            <string>Text Scale:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="qCTKSliderSpinBoxWidget" name="textScaleSliderSpinBoxWidget">
           <property name="maximum">
            <double>40.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="pointTab">
     <attribute name="title">
      <string>Point</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>18</y>
        <width>251</width>
        <height>236</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="qCTKColorPickerButton" name="pointUnselectedColorPickerButton">
           <property name="text">
            <string>Selected Point Color</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="qCTKColorPickerButton" name="pointSelectedColorPickerButton">
           <property name="text">
            <string>Unselected Line Color</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="glyphScaleLabel">
           <property name="text">
            <string>Point Size:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="opacityLabel">
           <property name="text">
            <string>Opacity:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="ambientLabel">
           <property name="text">
            <string>Ambient:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="diffuseLabel">
           <property name="text">
            <string>Diffuse:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="specularLabel">
           <property name="text">
            <string>Specular:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="qCTKSliderSpinBoxWidget" name="pointOpacitySliderSpinBoxWidget">
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="qCTKSliderSpinBoxWidget" name="pointAmbientSliderSpinBoxWidget">
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="qCTKSliderSpinBoxWidget" name="pointDiffuseSliderSpinBoxWidget">
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="qCTKSliderSpinBoxWidget" name="pointSpecularSliderSpinBoxWidget"/>
         </item>
         <item row="0" column="1">
          <widget class="qCTKSliderSpinBoxWidget" name="pointSizeSliderSpinBoxWidget"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <zorder>layoutWidget</zorder>
     <zorder>pointSizeSliderSpinBoxWidget</zorder>
    </widget>
    <widget class="QWidget" name="lineTab">
     <attribute name="title">
      <string>Line</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>19</y>
        <width>251</width>
        <height>236</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="qCTKColorPickerButton" name="lineUnselectedColorPickerButton">
           <property name="text">
            <string>Selected Line Color</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="qCTKColorPickerButton" name="lineSelectedColorPickerButton">
           <property name="text">
            <string>Unselected Line Color</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="widthLabel">
           <property name="text">
            <string>Line Width:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="qCTKSliderSpinBoxWidget" name="lineWidthSliderSpinBoxWidget">
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="opacityLabel_2">
           <property name="text">
            <string>Opacity:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="qCTKSliderSpinBoxWidget" name="lineOpacitySliderSpinBoxWidget">
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="ambientLabel_2">
           <property name="text">
            <string>Ambient:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="qCTKSliderSpinBoxWidget" name="lineAmbientSliderSpinBoxWidget">
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="diffuseLabel_2">
           <property name="text">
            <string>Diffuse:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="qCTKSliderSpinBoxWidget" name="lineDiffuseSliderSpinBoxWidget">
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="specularLabel_2">
           <property name="text">
            <string>Specular:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="qCTKSliderSpinBoxWidget" name="lineSpecularSliderSpinBoxWidget">
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="fiducialPropertyButtonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>620</y>
     <width>257</width>
     <height>28</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Modify Annotation Properties</string>
   </property>
  </widget>
  <widget class="qCTKCollapsibleGroupBox" name="CTKCollapsibleGroupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>281</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Display Annotation</string>
   </property>
   <widget class="QLabel" name="coordinatesLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>73</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Coordinates:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="annotationTypeBrowser">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>30</y>
      <width>259</width>
      <height>39</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>271</width>
      <height>39</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="annotationValueBrowser">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>140</y>
      <width>259</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>271</width>
      <height>29</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>76</y>
      <width>259</width>
      <height>63</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="textLabel">
       <property name="text">
        <string>Text:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="annotationTextEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qCTKCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qCTKCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qCTKColorPickerButton</class>
   <extends>QPushButton</extends>
   <header>qCTKColorPickerButton.h</header>
  </customwidget>
  <customwidget>
   <class>qCTKSliderSpinBoxWidget</class>
   <extends>QWidget</extends>
   <header>qCTKSliderSpinBoxWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qSlicermiAnnotationModule.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>fiducialPropertyButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>qSlicermiAnnotationModuleAnnotationPropertyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>600</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fiducialPropertyButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>qSlicermiAnnotationModuleAnnotationPropertyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>600</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
